cypress/pages/Registpage.js

```diff
@@ -2,20 +2,22 @@ export const Registpage = {
     TXT_FIRSTNAME: "#firstName",
     TXT_LASTNAME: "#lastName",
     TXT_EMAIL: "#userEmail",
-    RDO_GENDER: "#gender-radio-2",
+    RDO_GENDER: "#gender-radio-",
     TXT_MOBILE: "#userNumber",
-    DDL_DOB: "#dateOfBirthInput",
+    DDL_DOB: "#dateOfBirth",
```

Comment:
Why you change it? `#dateOfBirth` is the wrapper, the date picker is `#dateOfBirthInput`